microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -87,49 +279,292 @@ script.integrity = 'sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=';
+
+1. You then use the **Watch** or **Scope** pane, and use the controls at the top of the **Debugger** pane, such as **Step** (`F9`), to step through the code.
+
+<!-- "sources-paused-breakpoint.msft.png" is not currently directly used. It's the upstream source for a couple images, including the below image. -->
+
+:::image type="complex" source="../media/sources-paused-breakpoint-highlights.msft.png" alt-text="Entering Paused mode of the Debugger" lightbox="../media/sources-paused-breakpoint-highlights.msft.png":::
+   Entering Paused mode of the Debugger  
+:::image-end:::  
+
+The **Scope** pane shows variables that are in-scope, and their values.  In the above figure, we added the Watch expressions `sum` and `typeof sum`, and stepped two lines past the breakpoint.
+
+### Advantages of the debugger's Watch and Scope over console.log
+
+These three approaches are equivalent:
+
+*   Temporarily adding the statements `console.log(sum)` and `console.log(typeof sum)` in the code, where `sum` is in-scope.
+*   Issuing the statements `sum` and `console.log(typeof sum)` in the **Console** pane of the DevTools, when the debugger is paused where `sum` is in-scope.
+*   Setting the **Watch** expressions `sum` and `typeof sum` in the **Debugger** pane.
+
+When the variable `sum` is in-scope, `sum` and its value are automatically shown in the **Scope** section of the Debugger pane, and are also overlaid in the Editor pane where `sum` is calculated.  Therefore you probably wouldn't need to define a Watch expression for `sum`, or a `console.log` statement to show the value of `sum`.
+
+The debugger gives a richer, more flexible display and environment than a `console.log` statement.  For example, in the Debugger, as you step through the code, you may display and change the values of all currently defined properties and variables, and issue JavaScript statements in the **Console**.  (To display the Console, select **Esc**.)
+
+Breakpoints and Watch expressions are preserved when you refresh the page.
+
+### Articles about debugging
+
+The following articles cover the Debugger pane and breakpoints:
+
+*   [Get started with debugging JavaScript in Microsoft Edge DevTools][DevtoolsGuideChromiumJavascriptIndex] - A tutorial (with screen captures), using an existing, simple project.
+
+*   [Use the debugger features][DevToolsJavaScriptReference] - How to use the Debugger to set breakpoints, step through code, view and modify variable values, watch JavaScript expressions, and view the call stack (the sequence of function calls so far).
+
+*   [Pause your code with breakpoints][DevToolsJavaScriptBreakpoints] - How to set basic and specialized breakpoints in the debugger.
+
+## Extensions for Visual Studio Code, as an alternative to DevTools Sources tool
```

Comment:
I think we can cut this whole section. I don't think it adds much and we have our own docs for the VS Code extensions anyways